pkg/generated/openapi/def.bzl

```diff
@@ -0,0 +1,70 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_kubernetes_build//defs:go.bzl", "go_genrule")
+
+OPENAPI_TARGETS = [
+    "cmd/libs/go2idl/client-gen/test_apis/testgroup/v1",
```

Comment:
@thockin: After talking with @mikedanese, I think it's a good idea to automanage these lists with gazel.